feature/notice/src/main/java/com/yapp/feature/notice/notice/NoticeRoute.kt

```diff
@@ -163,6 +169,10 @@ fun NoticeScreen(
         }
 
     }
+
+    lazyScrollState.OnBottomReached {
+        onIntent.invoke(NoticeIntent.LoadMoreNoticeItem)
+    }
```

Comment:
```suggestion

lazyScrollState.OnBottomReached {
    onIntent(NoticeIntent.LoadMoreNoticeItem)
}
```

```kotlin
val lazyScrollState = rememberLazyListState()
lazyScrollState.OnBottomReached {
    onIntent(NoticeIntent.LoadMoreNoticeItem)
}
```

A: invoke를 굳이 붙이지 않아도 동작에는 차이가 없기 때문에, 특별한 이유가 없다면 생략해도 좋을 것 같습니다!
또한 코드 가독성 측면에서, lazyScrollState 선언 바로 아래에 OnBottomReached를 위치시키는 것도 좋을 것 같아요.

Reply:
저는 보통 .invoke 를 사용하는데 기존 로직을 보니 쓰지 않는게 많네요
[9ab9d40](https://github.com/YAPP-admin/yappu-world-android/pull/124/commits/9ab9d400bcb551a1ef1c450a378a5367a76d3871)

반영완료했습니다